### PR TITLE
Add domain to user translations

### PR DESCRIPTION
**Pull request by maukoquiroga**
partial_fix #77

### CODE REVIEW

File: priv/gettext/fr/LC_MESSAGES/default.po [hunk 1]
```diff
@@ -98,48 +98,3 @@ msgstr "En cliquant sur le bouton ci-dessous vous allez pouvoir créer un compte
 #: lib/transport_web/controllers/user_controller.ex:57
 msgid "connection_needed"
 msgstr "Vous devez être préalablement connecté·e."
```
Comment (l-vincent-l) [inline priv/gettext/fr/LC_MESSAGES/default.po]: You want to leave this one alone ?

Comment (maukoquiroga) [reply priv/gettext/fr/LC_MESSAGES/default.po]: Good catch!

File: priv/gettext/default.pot [hunk 2]
```diff
@@ -94,51 +94,3 @@ msgstr ""
 #: lib/transport_web/templates/page/login.html.eex:37
 msgid "login_what_is_going_to_happen"
```
Comment (l-vincent-l) [inline priv/gettext/default.pot]: you don't put this one in a domain?

Comment (maukoquiroga) [reply priv/gettext/default.pot]: Not in the scope of this PR, but I'll do in the next one.

Comment (maukoquiroga) [reply priv/gettext/default.pot]: Take a look at the description of #77

**Pull request merged**